ion-schema/src/model/schema_header.rs

```diff
@@ -36,10 +38,16 @@ impl SchemaHeader {
 }
 
 /// Represents an import in a [`SchemaHeader`].
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone)]
 pub struct Import {
     schema_id: String,
     type_import: Option<TypeImport>,
+    source_location: IslSourceLocation,
+}
+impl PartialEq for Import {
+    fn eq(&self, other: &Self) -> bool {
+        self.schema_id == other.schema_id && self.type_import == other.type_import
+    }
 }
```

Comment:
❗ `source_location` is extra metadata, not part of the semantic meaning of an `Import`, so we need a custom implementation of `PartialEq` to exclude it.